scripts/exgdas_atmos_nawips.sh

```diff
@@ -42,68 +39,49 @@ pdsext=no
 sleep_interval=10
 max_tries=180
 
-fhr=$(( 10#${fstart} ))
-while (( fhr <= 10#${fend} )); do
-  fhr3=$(printf "%03d" "${fhr}")
-
-  source_dirvar="COM_ATMOS_GRIB_${grid}"
-  GEMGRD="${RUN}_${grid}_${PDY}${cyc}f${fhr3}"
-  export GRIBIN="${!source_dirvar}/${model}.${cycle}.pgrb2.${grid}.f${fhr3}"
-  GRIBIN_chk="${GRIBIN}.idx"
-
-  if ! wait_for_file "${GRIBIN_chk}" "${sleep_interval}" "${max_tries}"; then
-    echo "FATAL ERROR: after 1 hour of waiting for ${GRIBIN_chk} file at F${fhr3} to end."
-    export err=7 ; err_chk
-    exit "${err}"
-  fi
-
-  cp "${GRIBIN}" "grib${fhr3}"
-
-  export pgm="nagrib2 F${fhr3}"
-  startmsg
-
-   ${NAGRIB} << EOF
-   GBFILE   = grib${fhr3}
-   INDXFL   = 
-   GDOUTF   = ${GEMGRD}
-   PROJ     = ${proj}
-   GRDAREA  = ${grdarea}
-   KXKY     = ${kxky}
-   MAXGRD   = ${maxgrd}
-   CPYFIL   = ${cpyfil}
-   GAREA    = ${garea}
-   OUTPUT   = ${output}
-   GBTBLS   = ${gbtbls}
-   GBDIAG   = 
-   PDSEXT   = ${pdsext}
-  l
-  r
+
+GEMGRD="${RUN}_${grid}_${PDY}${cyc}f${fhr3}"
+source_dirvar="COM_ATMOS_GRIB_${grid}"
+export GRIBIN="${!source_dirvar}/${model}.${cycle}.pgrb2.${grid}.f${fhr3}"
+GRIBIN_chk="${GRIBIN}.idx"
+
+if ! wait_for_file "${GRIBIN_chk}" "${sleep_interval}" "${max_tries}"; then
+  echo "FATAL ERROR: after 1 hour of waiting for ${GRIBIN_chk} file at F${fhr3} to end."
+  export err=7 ; err_chk
+  exit "${err}"
+fi
```

Comment:
Oops, missed this one when I did the gfs one. Same reasoning.
```suggestion
if [[ ! -r "${GRIBIN_chk}" ]]; then
  echo "FATAL ERROR: GRIB index file ${GRIBIN_chk} not found!"
  export err=7 ; err_chk
  exit "${err}"
fi
```